include/tvision/internal/platform.h

```diff
@@ -7,6 +7,7 @@
 #include <queue>
 #include <vector>
 #include <cstddef>
+#include <sys/param.h>
```

Comment:
What symbols does the `sys/parm.h` header provide that are needed in this file?

Reply:
This was for the `__FreeBSD__` symbol